proposals/2020/Add Operational Schedule Category.md

```diff
@@ -0,0 +1,12 @@
+# Update Annual Fuel Use Native Units
+
+## Overview
+
+This proposal is to add `Operational` as an enum choice for `auc:ScheduleCategory`
+
+## Justification
+
+211 6.2.1.1.e specifies
+> Schedules of occupant density, lighting, process and plug loads, and equipment use shall be provided for typical days
+
+There is no appropriate `auc:ScheduleCategory` for process and plug loads as well as equipment use. This new category would be intended to cover those cases.
```

Comment:
This covers my thoughts on the mapping from the excel doc:
- I think that plug loads can be mapped to the 'Miscellaneous equipment' enum.  See the BEDES definition https://bedes.lbl.gov/bedes-online/schedule-category.  Additionally, per the executive summary in this: https://www.nrel.gov/docs/fy12osti/51708.pdf, and through my conversations with others, plug and process loads are generally thought of as synonymous (with maybe plug loads being a subcategory of process loads, but generally they are pretty much the same)

So, that said:
1. I don't think we need this addition
1. To cover 6.2.1.1.e, I think we need to have schedule categories for:
    - 'Occupied', 'Lighting', 'Miscellaneous equipment', and 'HVAC equipment'.  'Heating equipment' and 'Cooling equipment' are both subcategories of 'HVAC equipment', but let's just be generic and call it 'HVAC equipment'.  The other potentially big one to consider from an end use consumption perspective would be 'Cooking equipment' as that can be big in dining / restaurants.

@nllong thoughts?